apps/vc-api/src/vc-api/credentials/credentials.service.spec.ts

```diff
@@ -187,39 +167,39 @@ describe('CredentialsService', () => {
       }
     });
     jest.spyOn(keyService, 'getPrivateKeyFromKeyId').mockResolvedValueOnce(key);
-    const vp = await service.provePresentation({ presentation, options: issuanceOptions });
-    expect(vc['proof']['jws']).toBeDefined();
+    const vp = await service.provePresentation({
+      presentation: rebeamPresentation,
+      options: issuanceOptions
+    });
+    expect(energyContractVerifiableCredential['proof']['jws']).toBeDefined();
     /**
      * Delete jws from proof as it is not deterministic
      * TODO: confirm this from the Ed25519Signature2018 spec
      */
     delete vp.proof.jws;
-    const expectedVpCopy = JSON.parse(JSON.stringify(expectedVp));
+    const expectedVpCopy = JSON.parse(JSON.stringify(vp));
     delete expectedVpCopy.proof.jws;
```

Comment:
Should be already deleted by line 179.

Reply:
I don't think so. `vp` and `expectedVpCopy` are different ojbects

Reply:
And why is that, since you are copying this object in line 180? 
`const expectedVpCopy = JSON.parse(JSON.stringify(vp));`

Reply:
Ah, you totally right. I looked at different place with similar check :+1: